docs/concepts/schedules.rst

```diff
@@ -95,6 +176,7 @@ Below is an example with duration in `days`.
 
 )
 
+.. _rate_unit:
 
 Rate Unit
```

Comment:
This is outside the scope of this PR, but I don't understand the difference between this section and the "Fixed rate schedules" section above.